### PR TITLE
[LEAF-1168] FE - add ui for Help Desk FAQ

### PR DESCRIPTION
**Pull request by g-theriot**

### CODE REVIEW

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 59
           

Complexity increasing per file
==============================
- LEAF_Request_Portal/faq/faqCategoryListManager.php  4
- LEAF_Request_Portal/faq/faqQuestions.php  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VALEAF/LEAF/pullRequest?prid=4692315&bid=15687115)

File: LEAF_Request_Portal/templates/menu.tpl [hunk 1]
```diff
@@ -1,37 +1,63 @@
...
-
-    $(menuButton).keypress(function(e) {
-        if (e.keyCode === 13) {
-            $(subMenu).css("display", "block");
-            $(menuButton).attr('aria-expanded', 'true');
-            subMenuButton.focus();
-        }
-    });
+    $(document).ready(function(){
+                var menuButton = $('#button_showLinks');
```
Comment (dastan-rahimi) [inline LEAF_Request_Portal/templates/menu.tpl]: Fix Indentation

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 59
           

Complexity increasing per file
==============================
- LEAF_Request_Portal/faq/faqCategoryListManager.php  4
- LEAF_Request_Portal/faq/faqQuestions.php  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VALEAF/LEAF/pullRequest?prid=4692315&bid=15687115)

File: LEAF_Request_Portal/templates/menu.tpl (hunk 1 above)
Comment (thomasnguyen95) [reply LEAF_Request_Portal/templates/menu.tpl]: Please fix the consistency with spacing/indentation across the files committed. Replace tab characters with 4 spaces.

**Pull request closed without merging**